.mailmap

```diff
@@ -44,8 +45,10 @@ Ben Noordhuis <info@bnoordhuis.nl> <bnoordhuis@bender.(none)>
 Ben Noordhuis <info@bnoordhuis.nl> <ben@strongloop.com>
 Ben Taber <ben.taber@gmail.com>
 Benjamin Coe <bencoe@gmail.com> <ben@npmjs.com>
+Benjamin Coe <bencoe@gmail.com> <bencoe@google.com>
 Benjamin Fleischer <github@benjaminfleischer.com> Benjamin Fleischer <benjamin.fleischer@swipesense.com>
-Benjamin Gruenbaum <inglor@gmail.com> <benji@peer5.com>
+Benjamin Gruenbaum <benjamingr@gmail.com> <benji@peer5.com>
+Benjamin Gruenbaum <benjamingr@gmail.com> <inglor@gmail.com>
```

Comment:
@benjamingr Is the new gmail one your preferred one? That’s what this is assuming here but please correct me if I’m wrong.

Reply:
Okay, then this is good here.